docs/runbooks/CDIDataImportCronOutdated.md

```diff
@@ -0,0 +1,66 @@
+# CDIDataImportCronOutdated
+
+## Meaning
+
+DataImportCron is in charge of recurring polling of latest version disk images as PVCs, commonly into a 'golden image' namespace;   
+These are VM Disk PVCs that always get updated with latest  version,  
+serving as a trustworthy clone source for created VMs (created from latest image of OS).  
+
+This alert fires when any DataImportCrons exist that fail to keep their corresponding PVCs updated (new `:latest` exists, we can't update to it).
+
+**Note**:  
+In the golden-image use case, `latest` is simply the latest OS version of the distribution.  
+In the non-golden-image case, as described above, we are merely referring to the latest hash of the image that is available.
+
+## Impact
+
+VMs are created from outdated disk images.
+
+## Diagnosis
+
+### Distinguish between golden images and regular crons
+DataImportCrons are widely used for golden images and likely serve as a source to create VM disks from, and thus it is vital for them to be up to date.
+- Find the erronous DataImportCron's namespace & name:
+	```bash
+	kubectl get dataimportcron -A -o json | jq -r '.items[] | select(.status.conditions[] | select(.type == "UpToDate" and .status == "False")) | .metadata.namespace + "/" + .metadata.name'
+	```
+	Output is returned as namespace/name - golden image crons will reside in the `openshift-virtualization-os-images` namespace.
+
+### Diagnosis artifacts
+- Substitute `<cron_namespace>`, `<cron_name>` to find the DataImportCron's corresponding DataVolume (resides in same namespace as cron):
```

Comment:
if using CNV auto-update mechanism, verify a default storage class is configured. One StorageClass should be marked with (default) next to its name:
```bash
    oc get storageclass
```

Reply:
I can add it to the Mitigation section but first want to be sure why this is a cnv-auto-update only issue

Reply:
I think this should be in both places as it's the first thing (and easiest one) a user should check, before fetching any other data from the cluster.
I added `if using CNV auto-update mechanism` but probably can be removed.

Reply:
done